ModTek/MergeCache.cs

```diff
@@ -132,7 +132,22 @@ public CacheEntry(string path, string originalPath, List<string> mergePaths)
                             continue;
                         }
 
-                        parentJObj.Merge(mergeJObj, new JsonMergeSettings { MergeArrayHandling = MergeArrayHandling.Replace });
+                        if (AdvancedJSONMerger.IsAdvancedJSONMerge(mergeJObj))
```

Comment:
I might flip the logic around on this so it's `IsNormalJsonMerge` or `!AdvancedJSONMerger.IsAdvancedJSONMerge(mergeJObj)` just so the common and easier to read case is presented first.

Reply:
I moved it so around that the standard behavior is still at the end but not in an else, so in case an advanced json merge failed (because its not really an instructions file), a standard merge should be tried none-the-less.